singularity-eos/eos/eos_base.hpp

```diff
@@ -836,6 +838,44 @@ class EosBase {
     PORTABLE_ALWAYS_THROW_OR_ABORT(msg);
   }
 
+  // JMM: This method is often going to be overloaded for special cases.
+  template <typename Indexer_t = Real *>
+  PORTABLE_INLINE_FUNCTION void
+  DensityEnergyFromPressureTemperature(const Real press, const Real temp,
+                                       Indexer_t &&lambda, Real &rho, Real &sie) const {
+    // TODO(JMM): A lot hardcoded in here... Hopefully relevent EOS's
+    // overwrite.
+    constexpr Real MAXFAC = 1e8;
+    constexpr Real EPS = 10 * robust::EPS();
+    constexpr Real MINR_DEFAULT = 1e-4;
```

Comment:
Is this a meaningful value? If not, I'd pick something more like 1e-08. That _should_ be low enough for most sesame tables.

Reply:
It's not meaningful. I think I can actually make these hardcoded constants go away by being a bit more clever.